client/header/activity-panel/panels/inbox/test/index.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
```

Comment:
Didn't we decide to use React Testing Library instead?

Reply:
You're right @jeffstieler. I removed `Enzyme` and added `React Testing Library` in the commit 68be4dc